# **Job Corps Programme**

**Author:**
[Anthony Strittmatter](http://www.anthonystrittmatter.com)

The Job Corps is the largest U.S. labor market program targeting disadvantaged youths.
It provides academic, vocational, and social training, as well as health care counseling and job
search assistance, for an average duration of eight to nine months. Mathematica Policy Research carried out a randomized experiment with the Job Corps. About 60% of the experimental participants were randomly selected to receive an offer to participate in the Job Corps. Out of these, 73\% joined the Job Corps program and
actually started to participate after an average duration of 1.4 months. The remaining experimental participants were assigned to the control group. Control group members were not eligible for the
Job Corps programs for three years following randomization (approximately 1\% participated
anyway).

We estimate the effects of receiving an offer to participate in the Job Corps on earnings fours years after the randomisation. We have access to 10,516 observations from the experimental Job Corps data.

**Variable list**

Outcome:
+ **EARNY**: Earnings per week in Year 4

Instrument:
+ **assignment**: Offer to join Job Corps

Treatment:
+ **participation**: Actual participation in Job Corps

Covariates (all dummies):
+ **female**: Dummy for females
+ **age_1**: Aged between 16-17 years
+ **age_2**: Aged between 18-19 years (omitted category)
+ **age_3**: Aged between 20-24 ye
+ **ed0_6**: 0-6 months education program in last year
+ **ed6_12**: 6-12 months education program in last yea
+ **hs_ged**: High school or GED credential
+ **white**: Dummy for white (omitted category)
+ **black**: Dummy for African-American
+ **hisp**: Dummy for Hispanics
+ **oth_eth**: Dummy for other ethnicity
+ **haschld**: Dummy for parents
+ **livespou**: Lives with spouse or partner
+ **everwork**: Ever had job for two weeks or more
+ **yr_work**: Worked in year prior to random assignment
+ **currjob**: Has job at random assignment
+ **job0_3**: Below 3 months employed in last year
+ **job3_9**: 3-9 months employed in last year
+ **job9_12**: 9-12 months employed in last year
+ **earn1**: Yearly earnings less than \$1,000
+ **earn2**: Yearly earnings \$1,000-5,000
+ **earn3**: Yearly earnings \$5,000-10,000
+ **earn4**: Yearly earnings above \$10,000
+ **badhlth**: Dummy for bad healt
+ **welf_kid**: Family on welfare when growing up
+ **got_fs**: Received food stamps in last year
+ **publich**: Public or rent-subsidized housing
+ **got_afdc**: Received AFDC in last year
+ **harduse**: Used hard drugs in last year
+ **potuse**: Smoked marijuana in last year
+ **evarrst**: Ever arrested dummy
+ **pmsa**: Lives in PMSA
+ **msa**: Lives in MSA

## Load Packages

In [20]:
########################  Load Packages  ########################

# List of required packages
pkgs <- c('fBasics', 'glmnet', 'dplyr', 'AER')

# Load packages
for(pkg in pkgs){
    library(pkg, character.only = TRUE)
}

print('All packages successfully installed and loaded.')

[1] "All packages successfully installed and loaded."


## Load Data

In [21]:
########################  Load Data Frame  ########################
#set.seed(100239) # set starting value for random number generator

# Load data frame
df <- read.csv("Data/job_corps.csv",header=TRUE, sep=",")
df <- df[,-c(6,11)] # delete omitted categories

print('Data successfully loaded.')

[1] "Data successfully loaded."


## Means and Standard Deviations

In [22]:
########################  Table with Descriptive Statistics  ########################
desc <- fBasics::basicStats(df) %>% t() %>% as.data.frame() %>% 
  select(Mean, Stdev, Minimum, Maximum, nobs)
print(round(desc, digits=2))

                Mean  Stdev Minimum Maximum  nobs
EARNY4        204.44 195.69       0 2409.91 10516
assignment      0.60   0.49       0    1.00 10516
participation   0.44   0.50       0    1.00 10516
female          0.43   0.49       0    1.00 10516
age_1           0.41   0.49       0    1.00 10516
age_3           0.27   0.45       0    1.00 10516
ed0_6           0.26   0.44       0    1.00 10516
ed6_12          0.36   0.48       0    1.00 10516
hs_ged          0.24   0.43       0    1.00 10516
black           0.49   0.50       0    1.00 10516
hisp            0.17   0.38       0    1.00 10516
oth_eth         0.07   0.26       0    1.00 10516
haschld         0.20   0.40       0    1.00 10516
livespou        0.06   0.24       0    1.00 10516
everwork        0.80   0.40       0    1.00 10516
yr_work         0.64   0.48       0    1.00 10516
currjob         0.21   0.40       0    1.00 10516
job0_3          0.22   0.41       0    1.00 10516
job3_9          0.30   0.46       0    1.00 10516


## Unconditional OLS regression

In [23]:
### Unconditional OLS Regression ###

ols <- lm(EARNY4 ~ participation, data = df)
summary(ols)


Call:
lm(formula = EARNY4 ~ participation, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-211.44 -168.41  -25.03  101.08 2210.97 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept)    198.936      2.550  78.023  < 2e-16 ***
participation   12.503      3.842   3.254  0.00114 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 195.6 on 10514 degrees of freedom
Multiple R-squared:  0.001006,	Adjusted R-squared:  0.0009111 
F-statistic: 10.59 on 1 and 10514 DF,  p-value: 0.001141


Results suggest that participation in Job Corps increases weekly earnings by 12.50 US-dollars. Possibly we have selection bias, because experimental participants with an offer can self-select into actual participation.

## Conditional OLS Regression

In [24]:
### Conditional OLS Regression ###

ols <- lm(EARNY4 ~ ., data = df[,-2])
summary(ols)


Call:
lm(formula = EARNY4 ~ ., data = df[, -2])

Residuals:
    Min      1Q  Median      3Q     Max 
-384.12 -131.53  -19.42   91.17 2192.45 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept)   195.6631     8.0832  24.206  < 2e-16 ***
participation  15.3066     3.6442   4.200 2.69e-05 ***
female        -62.4533     3.9727 -15.721  < 2e-16 ***
age_1          -4.4015     4.6782  -0.941 0.346800    
age_3           8.7360     5.1515   1.696 0.089949 .  
ed0_6           9.2106     4.7476   1.940 0.052401 .  
ed6_12         12.1485     4.7915   2.535 0.011245 *  
hs_ged         29.1206     4.8865   5.959 2.61e-09 ***
black         -56.6921     4.8168 -11.770  < 2e-16 ***
hisp          -30.9087     5.8729  -5.263 1.45e-07 ***
oth_eth       -27.9901     7.7033  -3.634 0.000281 ***
haschld        12.4896     5.2846   2.363 0.018128 *  
livespou       -3.6477     7.6944  -0.474 0.635460    
everwork       12.6284     6.2326   2.026 0.042772 *  
yr_work        48

Results suggest that participation in Job Corps increases weekly earnings by 15.31 US-dollars. This suggests the unconditional OLS estimates are negatively biased. But is controlling for observable characteristics sufficient to eliminate the entire selection bias of OLS?

## Unconditional Instrumental Variable Approach

In [19]:
### Unconditional IV Regression ###

iv <- ivreg(formula = EARNY4 ~ participation | assignment, data = df)
summary(iv)


Call:
ivreg(formula = EARNY4 ~ participation | assignment, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-213.84 -168.37  -23.14  100.57 2212.87 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept)    197.042      3.058  64.434  < 2e-16 ***
participation   16.803      5.428   3.096  0.00197 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 195.6 on 10514 degrees of freedom
Multiple R-Squared: 0.0008871,	Adjusted R-squared: 0.0007921 
Wald test: 9.584 on 1 and 10514 DF,  p-value: 0.001968 


Results suggest that participation in Job Corps increases weekly earnings by 16.80 US-dollars. Accordingly, even the conditional OLS results are negatively biased.

## Conditional Instrumental Variable Approach

In [28]:
### Conditional IV Regression ###

iv <- ivreg(formula = EARNY4 ~ participation + female + age_1 + age_3 + ed0_6 + ed6_12 + hs_ged + black + hisp + oth_eth + haschld + livespou + everwork + yr_work + currjob + job0_3 + job3_9 + job9_12 + earn1 + earn2 + earn3 + earn4 + badhlth + welf_kid + got_fs + publich + got_afdc + harduse + potuse + evarrst + pmsa + msa | assignment + female + age_1 + age_3 + ed0_6 + ed6_12 + hs_ged + black + hisp + oth_eth + haschld + livespou + everwork + yr_work + currjob + job0_3 + job3_9 + job9_12 + earn1 + earn2 + earn3 + earn4 + badhlth + welf_kid + got_fs + publich + got_afdc + harduse + potuse + evarrst + pmsa + msa, data = df)
summary(iv)


Call:
ivreg(formula = EARNY4 ~ participation + female + age_1 + age_3 + 
    ed0_6 + ed6_12 + hs_ged + black + hisp + oth_eth + haschld + 
    livespou + everwork + yr_work + currjob + job0_3 + job3_9 + 
    job9_12 + earn1 + earn2 + earn3 + earn4 + badhlth + welf_kid + 
    got_fs + publich + got_afdc + harduse + potuse + evarrst + 
    pmsa + msa | assignment + female + age_1 + age_3 + ed0_6 + 
    ed6_12 + hs_ged + black + hisp + oth_eth + haschld + livespou + 
    everwork + yr_work + currjob + job0_3 + job3_9 + job9_12 + 
    earn1 + earn2 + earn3 + earn4 + badhlth + welf_kid + got_fs + 
    publich + got_afdc + harduse + potuse + evarrst + pmsa + 
    msa, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-387.80 -131.21  -20.11   91.28 2195.40 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept)   192.8675     8.2348  23.421  < 2e-16 ***
participation  21.7253     5.1204   4.243 2.23e-05 ***
female        -62.5575     3.9737 -15.743  

Results suggest that participation in Job Corps increases weekly earnings by 21.73 US-dollars. This suggest the instrument is confounded by the observed characteristics (and possibly also unobserved characteristics). It seems that the randomisation did not work approriately. This might be an explanation why 1\% of the experimental participants could participate in Job Corps without an offer to participate.